src/views/CheckoutView.vue

```diff
@@ -13,18 +13,23 @@
 </template>
 
 <script>
-import Events from '../Events/Events.vue';
+import EventsListScroll from '../components/Events/EventsListScroll.vue';
 
 export default {
   name: 'Checkout',
   components: {
-    Events,
+    EventsListScroll,
+  },
+  computed: {
+    registeredEvents() {
```

Comment:
How is this going to be used in the future? Will it be used to fetch Events from the store that this user is registered for?

Reply:
It will be held in the store because register events will be held in browser until they are purchased.